libvast/builtins/endpoints/export.cpp

```diff
@@ -202,17 +460,38 @@ export_multiplexer_actor::behavior_type export_multiplexer(
         rq.response->abort(400, "couldn't parse expression\n");
         return;
       }
-      constexpr size_t DEFAULT_EXPORT_LIMIT = 50;
-      size_t limit = DEFAULT_EXPORT_LIMIT;
+      auto params = export_parameters{
+        .expr = std::move(*expr),
+      };
+      // Adjust defaults for the `/export/
+      if (endpoint_id == ENDPOINT_EXPORT_TYPED) {
+        params.format_opts.typed_results = true;
+        params.format_opts.flatten = true;
+      }
       if (rq.params.contains("limit")) {
         auto& param = rq.params.at("limit");
         // Should be type-checked by the server.
         VAST_ASSERT(caf::holds_alternative<count>(param));
-        limit = caf::get<count>(param);
+        params.limit = caf::get<count>(param);
+      }
+      if (rq.params.contains("flatten")) {
+        auto& param = rq.params.at("flatten");
+        VAST_ASSERT(caf::holds_alternative<bool>(param));
+        params.format_opts.flatten = caf::get<bool>(param);
+      }
```

Comment:
Isn't something a little more concise possible as well?
```suggestion
      params.format_opts.flatten = caf::get_or(rq.params, "flatten", params.format_opts.flatten);
```